pkg/core/config/defaults.go

```diff
@@ -0,0 +1,81 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+	"github.com/updatecli/updatecli/pkg/core/pipeline/scm"
+)
+
+// GetChangelogTitle try to guess a specific target based on various information available for
+// a specific job
+func (config *Config) GetChangelogTitle(ID string, fallback string) (title string) {
+	if len(config.Title) > 0 {
+		// If a pipeline title has been defined, then use it for pull request title
+		title = fmt.Sprintf("[updatecli] %s",
```

Comment:
The `[updatecli]` prefix is already harcoded like that elsewhere?

Reply:
I've only changed the method from one filte to the other, I won't touch this method on this PR.

But that is a good question that you raise though :)

Reply:
> The `[updatecli]` prefix is already harcoded like that elsewhere?

In the same function yes 😆 , that function could be better written but well 🤷🏾